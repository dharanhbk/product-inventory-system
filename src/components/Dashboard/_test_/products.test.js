import React from 'react';
import ReactDOM from 'react-dom'
import {MemoryRouter as Router} from 'react-router-dom'
import {configure,shallow} from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import '@testing-library/jest-dom/extend-expect'
import  Product from '../product'

configure({adapter:new Adapter()});


describe("all test for Products component", ()=>{
    it('renders the Products component without crashing!', ()=>{
      const div = document.createElement('div')
      ReactDOM.render(<Router><Product></Product></Router>, div)
    })
  })

  test('should test Products component with default state of empty array', () => {
    const wrapper = shallow(<Product product={[]} />);
    expect(wrapper).toMatchSnapshot();
   });