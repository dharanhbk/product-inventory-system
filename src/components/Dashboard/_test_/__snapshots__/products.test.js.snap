// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test Products component with default state of empty array 1`] = `ShallowWrapper {}`;
